src/test/java/jewellery/inventory/unit/service/ProductServiceTest.java

```diff
@@ -328,7 +327,7 @@ void testUpdateProductSuccessfullyWhenProductIsPartOfAnotherProduct() {
     productService.updateProduct(innerProduct.getId(), productRequestDto);
 
     verify(productRepository, times(1)).findById(innerProduct.getId());
-    verify(userRepository,times(2)).findById(user.getId());
```

Comment:
We were looking for a user in the database because we needed the user to be passed to setProductField and since we changed it to product.owner() we no longer call it 2 times.